README.md

```diff
@@ -125,6 +125,7 @@ EOF
 
 #### Save kubeconfig
 ```sh
+mv ~/.kube/config ~/.kube/config_bkp
```

Comment:
Lets us use proper English and write backup ;-)

Reply:
Done